### PR TITLE
[PHP] Added hyperf framework.

### PR DESCRIPTION
**Pull request by limingxinleo**
Before submitting your PR, please review the following checklist :

## If you are adding a framework

+ [x] A `Dockerfile` exists, or can be built ?
+ [x] I've updated `CI` config file (and **commited** `.travis.yml`)
~~~sh
bin/make ci_config
~~~
+ [x] I've updated `.dependabot` config file  (and **commited** `.dependabot/config.yml`)
~~~sh
bin/make deps_config
~~~
+ [x] All tests passes ?
~~~
export FRAMEWORK=<MY_FRAMEWORK>
shards install
shards build --release
bin/make config
bin/neph ${FRAMEWORK}
crystal spec
~~~

### CODE REVIEW

**Comment by waghanza** [comment]
@huangzhhui OK to include `hyperf` in here ?

**Comment by huangzhhui** [comment]
> @huangzhhui OK to include `hyperf` in here ?

Yes, definitely OK

**Comment by waghanza** [comment]
@limingxinleo where is the file `/opt/www/bin/hyperf.php` mentionned on **docker** ?

File: php/hyperf/swoole.ini [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[swoole]
```
Comment (waghanza) [inline php/hyperf/swoole.ini]: no need
this file is already created by `docker-php-ext-enable swoole` in *Dockerfile**

Comment (huangzhhui) [reply php/hyperf/swoole.ini]: @waghanza Hyperf have to set `swoole.use_shortname` to `'Off'`

Comment (huangzhhui) [reply php/hyperf/swoole.ini]: @waghanza Hyperf have to set swoole.use_shortname to 'Off'

**Comment by limingxinleo** [comment]
All test passed, could you review it again?

**Comment by waghanza** [comment]
@limingxinleo sure, could you remove unused files (as @inhere does for `swoft`) ? And I'll merge it

**Comment by huangzhhui** [comment]
> @limingxinleo sure, could you remove unused files (as @inhere does for `swoft`) ? And I'll merge it

All useless files or code have been removed. Please review again

**Comment by huangzhhui** [comment]
@waghanza I noticed that the results still does not has Hyperf in the list, How or When the results will be re-generate ?

**Pull request merged**